src/style.css

```diff
@@ -0,0 +1,272 @@
+  content: "11";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day12::before {
+  content: "12";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day13::before {
+  content: "13";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day14::before {
+  content: "14";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day15::before {
+  content: "15";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day16::before {
+  content: "16";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day17::before {
+  content: "17";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day18::before {
+  content: "18";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day19::before {
+  content: "19";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day20::before {
+  content: "20";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day21::before {
+  content: "21";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day22::before {
+  content: "22";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day23::before {
+  content: "23";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day24::before {
+  content: "24";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day25::before {
+  content: "25";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day26::before {
+  content: "26";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day27::before {
+  content: "27";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day28::before {
+  content: "28";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day29::before {
+  content: "29";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day30::before {
+  content: "30";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day31::before {
+  content: "31";
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day--mon {
+  margin-left: 0;
+}
+
+.day--tue {
+  margin-left: 105px;
+}
+
+.day--wed {
+  margin-left: 205px;
+}
+
+.day--thu {
+  margin-left: 305px;
+}
+
+.day--fri {
+  margin-left: 405px;
+}
+
+.day--sat {
+  margin-left: 505px;
+}
+
+.day--sun {
+  margin-left: 606px;
+}
+
+.calendar .day__month--length:nth-child(28) {
+  opacity: 1;
+}
+
+.calendar .day__month--length:nth-child(29) {
+  opacity: 1;
+}
+
+.calendar .day__month--length:nth-child(30) {
+  opacity: 1;
+}
+
+.calendar .day__month--length:nth-child(31) {
+  opacity: 1;
+}
```

Comment:
The use of nth-child selectors to control the visibility of days based on month length assumes a fixed HTML structure. If the order of elements changes, this could lead to incorrect styling. Consider using classes or data attributes to manage month-specific styles more reliably.